src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
--->
-        <!-- RAB 2014/08/18 Should move flush right with some vertical spacing -->
-<!--        <attribution>Thomas W. Judson <br />Nacogdoches, Texas 2015</attribution>
-
-    </preface>-->
+    <acknowledgement>
+        <p>Over several terms, Jack Green wrote most of the book's content as a Word/PDF document for use in his face-to-face and online Precalculus courses at Mt. Hood Community College.</p>
+        <p>In 2016/2017, Nick Chura, Jack Green and Dr. Alex Jordan converted the material to MathBook XML (enabling the HTML output alongside a PDF) with support from an Open Oregon grant.</p>
+        <p>Special thanks to Dr. Rob Beezer, David Favreault, Jim Hatton, Robert Hauss, Dr. Alex Jordan, Dr. Elise Lockwood and Gina Shankland for proofreading and review.</p>
+    </acknowledgement>
+
+    <preface xml:id="to-all">
+        <title>To All</title>
+        <p>Precalculus is taught at Mt. Hood Community College over two <m>10</m><ndash />week courses. This text aims to address all learning outcomes from the first course, suitable for use in a single <m>10</m> week term.</p>
+
+        <paragraphs>
+            <title>HTML and PDF</title>
+            <p>This text will be released with several synchronous versions that offer different features. The essential content of each version is the same as for all others.</p>
+            <ul>
+                <li><p>Currently, a preview of the web version is available at <url href="http://www.greenprecalc.com/Math%20111/precalc1-MHCC/html/"/>. The web version offers full walk-through solutions to most exercises, as well as interactive elements and easier navigation than print versions.</p></li>
+                <li><p>A PDF-for-print will be available in June 2017. This version is designed to be printed, not read on a screen. To save on printing expense, this version is mostly black-and-white.</p></li>
+            </ul>
+        </paragraphs>
+
+        <paragraphs>
+            <title>Copying Content</title>
+
+            <p>Mathematical content can be copied from the HTML version. To copy math content into MS Word, right-click or control-click over the math content, and click to Show Math As MathML Code. Copy the resulting code, and Paste <em>Special</em> into Word. In the Paste Special menu, paste it as Unformatted Text. To copy math content into <latex/> source, right-click or control-click over the math content, and click to Show Math As TeX Commands.</p>
```

Comment:
Is this what you're suggesting?

`<paragraphs>
            <title>Copying Content</title>
            <p>Mathematical content can be copied from the HTML version.</p>
            <p>To copy math content into MS Word:
            <c>Right-click or control-click over the math content</c>
            <c>Show Math As MathML Code</c>
            <c>Paste Special</c>
            <c>Unformatted Text</c></p>
            <p>To copy math content into <latex/> source:
            <c>Right-click or control-click over the math content</c>
            <c>Show Math As TeX Commands</c></p>
        </paragraphs>`